install/mysql/plugin_flyvemdm_empty.sql

```diff
@@ -16,6 +16,7 @@ CREATE TABLE IF NOT EXISTS `glpi_plugin_flyvemdm_agents` (
   `plugin_flyvemdm_fleets_id` int(11)                                           DEFAULT NULL,
   `last_report`               datetime                                          DEFAULT NULL,
   `last_contact`              datetime                                          DEFAULT NULL,
+  `online`                    tinyint(1)                                        NOT NULL DEFAULT '0',
```

Comment:
this is a boolean, and in GLPI a bool is encoded with a 1 or a 0 in a single digit column